filebeat/docs/configuration.asciidoc

```diff
@@ -150,48 +151,59 @@ The buffer size every harvester uses when fetching the file. The default is 1638
 
 ===== max_bytes
 
-Maximum number of bytes a single log event can have. All bytes after max_bytes are discarded and not sent.
-This is especially useful for multiline log messages which can get large. The default is 10MB (10485760).
+The maximum number of bytes that a single log event can have. All bytes after `max_bytes` are discarded and not sent.
+This setting is especially useful for multiline log messages, which can get large. The default is 10MB (10485760).
 
 ===== multiline
 
-Mutiline can be used for log messages spanning multiple lines. This is common for Java Stack Traces
-or C-Line Continuation. The following example combines all lines following a line start with a `[`.
-This could for example be a Java Stack Trace.
+Options that control how Filebeat deals with log messages that span multiple lines. Multiline messages are common in Java stack traces and C line continuation. 
+
+The following example shows how to configure Filebeat to handle a multiline message where the first line of the message begins with a bracket (`[`). 
```

Comment:
@ruflin That's a good question. I looked at the Logstash doc when I was working the edits, but I felt that the structure of their doc is so different from our doc that trying to follow their model wouldn't really work for us right now. Also, the language they use in their descriptions is not very precise. For example, they say: "The pattern should match what you believe to be an indicator that the field is part of a multi-line event." This isn't strictly true because not every line in the multiline message matches the pattern depending on what you've specified for other options. So rather than be consistently confusing, I decided to come up with my own descriptions that (I hope) are more precise. Also, because we've renamed a couple of the options, the Logstash descriptions wouldn't make sense in our doc. 

Examples are definitely key here. Can you point me to the tests where we have some examples that I can use?